src/sonic-yang-models/yang-models/sonic-vxlan.yang

```diff
@@ -0,0 +1,212 @@
+module sonic-vxlan {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-vxlan";
+    prefix svxlan;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+    import sonic-vlan {
+        prefix svlan;
+    }
+	import sonic-types {
```

Comment:
Please correct alignment

Reply:
The reason is mixing tab and spaces. So it seems aligned in some editor.

Reply:
Removed a few tabs which were present.